config_parser/config.go

```diff
@@ -20,10 +20,11 @@ type (
 	}
 
 	GenoMagicConfig struct {
-		InputFilePath string `yaml:"inputFilePath"`
-		OutputPath    string `yaml:"outputPath"`
-		Threads       int    `yaml:"threads"`
-		Prep          bool   `yaml:"prep"`
+		Assemblers    []string `yaml:"assemblers,flow"`
```

Comment:
`flow`?

Reply:
`flow` style is the flag that GoDoc suggested to use for collection items such as arrays.

https://godoc.org/gopkg.in/yaml.v2
<img width="717" alt="image" src="https://user-images.githubusercontent.com/19979068/83958769-de527b00-a832-11ea-886a-4db94862ff47.png">